ui/admin/app/controllers/scopes/scope/workers/index.js

```diff
@@ -19,8 +20,46 @@ export default class ScopesScopeWorkersIndexController extends Controller {
 
   // =attributes
 
+  queryParams = [
+    'search',
+    { releaseVersions: { type: 'array' } },
+    'page',
+    'pageSize',
+  ];
+
+  @tracked search;
+  @tracked releaseVersions = [];
+  @tracked page = 1;
+  @tracked pageSize = 10;
   @tracked selectedWorker;
 
+  /**
+   * Get release_versions for filter
+   * @type {object[]}
+   */
+  get releaseVersionOptions() {
+    const releaseVersions = this.model.workers
```

Comment:
I think you might need to get these from all workers as if you pull it from just `workers`, searching will change what filters are available

Reply:
Yep, you are right. I will update that now.